tests/test_compressors/model_compressors/test_model_compressor.py

```diff
@@ -443,9 +441,7 @@ def test_compress_model(model_stub, q_format, s_config, tmpdir):
 )
 def test_compress_model_meta(model_stub, q_format, s_config):
     # Load model on CPU to get expected compressed state_dict
-    cpu_model = AutoModelForCausalLM.from_pretrained(
-        model_stub, torch_dtype=torch.float32
```

Comment:
Why does the dtype change? I'm good with the change, I just want to make sure that `torch.float32` is still supported